src/task.rs

```diff
@@ -1165,6 +1171,13 @@ pub struct TaskMetrics {
     /// ```
     pub total_fast_poll_duration: Duration,
 
+    /// The total scheduling duration for tasks which were quickly executed after being scheduled.
+    ///
+    /// Here, "quickly" means that the task takes less than
+    /// [`long_delay_threshold`][TaskMonitor::long_delay_threshold] to be executed after being
+    /// scheduled.
```

Comment:
Can you follow the documentation pattern of [`total_fast_poll_duration`](https://docs.rs/tokio-metrics/0.2.0/tokio_metrics/struct.TaskMetrics.html#structfield.total_fast_poll_duration) here?